.github/ISSUE_TEMPLATE/5.priority-support.yml

```
@@ -1,5 +1,5 @@
-name: 'Priority Support: SLA ⏰'
-description: I'm an MUI X Premium user and we have purchased the Priority Support add-on. I can't find a solution to my problem with MUI X.
+name: 'Priority support: SLA ⏰'
```

```suggestion
Priority Support: SLA ⏰
```

To match https://mui.com/legal/technical-support-sla/#priority-support?